docs/aws/audit/sesmonitoring/rules/is_identity_exposed.mdx

```diff
@@ -23,6 +23,283 @@ CBP
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. Install and configure AWS CLI: Before you can start using AWS CLI, you need to install it on your local machine and configure it with your AWS account credentials. You can do this by running the following commands:
+
+   Installation:
+   ```
+   pip install awscli
+   ```
+   Configuration:
+   ```
+   aws configure
+   ```
+   You will be prompted to provide your AWS Access Key ID, Secret Access Key, Default region name, and Default output format.
+
+2. List all SES identities: Once your AWS CLI is set up, you can list all the SES identities in your account by running the following command:
+   ```
+   aws ses list-identities
+   ```
+   This command will return a list of all identities (email addresses and domains) that are associated with your AWS account.
+
+3. Get identity policies: For each identity, you can get the policy associated with it by running the following command:
+   ```
+   aws ses get-identity-policies --identity "identity-name"
+   ```
+   Replace "identity-name" with the name of the identity you want to check. This command will return the policy document associated with the specified identity.
+
+4. Check the policy: The policy document returned in the previous step will contain information about who has what permissions for the specified identity. You need to manually check this policy to see if it exposes the identity to the public. If the policy allows actions from a principal that is set to "*", this means the identity is exposed to the public.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Install the necessary AWS SDK for Python (Boto3) if you haven't done so already. You can do this by running the command `pip install boto3` in your terminal.
+
+2. Import the necessary modules and establish a session with AWS using your credentials. Here's a basic setup:
+
+```python
+import boto3
+
+session = boto3.Session(
+    aws_access_key_id='YOUR_ACCESS_KEY',
+    aws_secret_access_key='YOUR_SECRET_KEY',
+    region_name='us-west-2'  # or any other region you prefer
+)
+```
+
+3. Now, create a client for the SES service:
+
+```python
+ses_client = session.client('ses')
+```
+
+4. Use the `list_identities()` method to retrieve all identities (email addresses and domains) associated with your AWS account. Then, check if any of these identities are exposed:
+
+```python
+def check_ses_identities_exposure():
+    response = ses_client.list_identities()
+    identities = response['Identities']
+    
+    for identity in identities:
+        attributes = ses_client.get_identity_attributes(Identity=identity)
+        if 'ForwardingEnabled' in attributes['Attributes']:
+            if attributes['Attributes']['ForwardingEnabled'] == 'true':
+                print(f"Identity {identity} is exposed.")
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>